documentation/attributes.html

```diff
@@ -41,47 +26,471 @@
+    <td>ROF</td>
+    <td>FW for singlevalue numeric attributes and MIW for
+      multivalue attributes. EIW for singlevalue string or if the
+      attribute is singlevalue fast-search</td>
+  </tr>
+  <tr>
+    <td>Multivalue mapping</td>
+    <td>D * V * ((FW or EIW) + WW)</td>
+    <td>ROF</td>
+    <td>Applicable only for array or weighted sets. EIW if string or fast-search</td>
+  </tr>
+  <tr>
+    <td>Enum store</td>
+    <td>U * ((FW + VW) + 4 + ((EIW + PIW) or EIW))</td>
+    <td>ROF</td>
+    <td>Applicable for strings or if fast-search is set.
+      (EIW + PIW) if fast-search is set, EIW otherwise.</td>
+  </tr>
+  <tr>
+    <td>Posting list</td>
+    <td>D * V * PW</td>
+    <td>ROF</td>
+    <td>Applicable if fast-search is set</td>
+  </tr>
+  </tbody>
+</table>
+
+
+<h3 id="variants">Variants</h3>
+<table class="table">
+  <thead>
+  <tr>
+    <th>Type</th>
+    <th>Components</th>
+    <th>Formula</th>
+  </tr>
+  </thead>
+
+  <tbody>
+  <tr>
+    <td>Numeric singlevalue plain</td>
+    <td>Document vector</td>
+    <td>D * FW * ROF</td>
+  </tr>
+  <tr>
+    <td>Numeric multivalue value plain</td>
+    <td>Document vector, Multivalue mapping</td>
+    <td>D * MIW * ROF + D * V * (FW+WW) * ROF </td>
+  </tr>
+  <tr>
+    <td>Numeric singlevalue fast-search</td>
+    <td>Document vector, Enum store, Posting List</td>
+    <td>D * EIW * ROF + U * (FW+4+EIW+PIW) * ROF + D * PW * ROF</td>
+  </tr>
+  <tr>
+    <td>Numeric multivalue value fast-search</td>
+    <td>Document vector, Multivalue mapping, Enum store, Posting List</td>
+    <td>D * MIW * ROF + D * V * (EIW+WW) * ROF + U * (FW+4+EIW+PIW) * ROF + D * V * PW * ROF</td>
+  </tr>
+  <tr>
+    <td>Singlevalue string plain</td>
+    <td>Document vector, Enum store</td>
+    <td>D * EIW * ROF + U * (FW+VW+4+EIW) * ROF</td>
+  </tr>
+  <tr>
+    <td>Singlevalue string fast-search</td>
+    <td>Document vector, Enum store, Posting List</td>
+    <td>D * EIW * ROF + U * (FW+VW+4+EIW+PIW) * ROF + D * PW * ROF</td>
+  </tr>
+  <tr>
+    <td>Multivalue string plain</td>
+    <td>Document vector, Multivalue mapping, Enum store</td>
+    <td>D * MIW * ROF + D * V * (EIW+WW) * ROF + U * (FW+VW+4+EIW) * ROF</td>
+  </tr>
+  <tr>
+    <td>Multivalue string fast-search</td>
+    <td>Document vector, Multivalue mapping, Enum store, Posting list</td>
+    <td>D * MIW * ROF + D * V * (EIW+WW) * ROF + U * (FW+VW+4+EIW+PIW) * ROF + D * V * PW * ROF</td>
+  </tr>
+  <tr>
+    <td>Boolean singlevalue</td>
+    <td>Document vector</td>
+    <td>D * FW * ROF</td>
+  </tr>
+  </tbody>
+</table>
+
+
+
+<h2 id="multivalue-mapping-enum-store">Multivalue Mapping and Enum Store</h2>
 <p>
-<a href="reference/schema-reference.html#attribute">fast-access</a> locks all replicas of the attribute in memory.
-See <a href="performance/sizing-feeding.html">sizing feeding</a> for details.
+  The attribute Multivalue Mapping and Enum Store has upper limits, limiting the number of values in an attribute.
```

Comment:
Consider a sentence explaining what these components are used for: multivalue mapping storing arrays of values for array and weighed set types, and enum store storing all unique strings for all string attributes.

Reply:
yup! left as a todo for now